google/cloud/logging_v2/handlers/handlers.py

```diff
@@ -213,6 +215,11 @@ def emit(self, record):
         labels = {**add_resource_labels(resource, record), **(labels or {})} or None
 
         # send off request
+        if self.transport is None:
+            self.transport = self._transport_cls(
+                self.client, self.name, resource=self.resource
+            )
```

Comment:
Wouldn't this mean the handler has been closed? Why not just raise an error, instead of re-creating the transport?

Reply:
I based this off of some logging handlers I saw in the Python standard library, which will reopen a closed connection if that connection is closed.

Reply:
Personally I think this adds extra complexity, so I'd prefer to keep them permanently closed if we can get away with that

But if you think this behavior would help us integrate with the standard library, it makes sense to keep it.